CHANGELOG.md

```diff
@@ -1,8 +1,117 @@
+  Implement the optional `replacer` parameter in `JSON.stringify( value[, replacer [, space] ] )` (@n14little)
+- [FEATURE #480](https://github.com/boa-dev/boa/issues/480):
+  Implement global `Infinity` property (@AnirudhKonduru)
+- [FEATURE #410](https://github.com/boa-dev/boa/pull/410):
+  Add support for the reviver function to JSON.parse (@abhijeetbhagat)
+- [FEATURE #425](https://github.com/boa-dev/boa/pull/425):
+  Specification compliant `ToString` (`to_string`) (@HalidOdat)
+- [FEATURE #442](https://github.com/boa-dev/boa/pull/442):
+  Added `TypeError` implementation (@HalidOdat)
+- [FEATURE #296](https://github.com/boa-dev/boa/issues/296):
+  using measureme for profiling the interpreter (@jasonwilliams)
+- [FEATURE #450](https://github.com/boa-dev/boa/pull/450):
+  Specification compliant `ToBigInt` (`to_bigint`) (@HalidOdat)
+- [FEATURE #422](https://github.com/boa-dev/boa/pull/422):
+  Move AST Nodes into their own files, dividing the run function (@Lan2u) (@Razican)
+- [FEATURE #455](https://github.com/boa-dev/boa/pull/455):
+  TemplateLiteral Basic lexer implementation (@croraf)
+- [FEATURE #447](https://github.com/boa-dev/boa/issues/447):
+  parseInt, parseFloat implementation (@Lan2u)
+- [FEATURE #435](https://github.com/boa-dev/boa/issues/435):
+  Optimize type comparisons (@Lan2u)
+- [FEATURE #468](https://github.com/boa-dev/boa/pull/468):
+  Add BigInt.asIntN() and BigInt.asUintN() functions (@Tropid)
+- [FEATURE #428](https://github.com/boa-dev/boa/issues/428):
+  [Feature Request] - Create benchmark for Array manipulation (@abhijeetbhagat)
+- [FEATURE #439](https://github.com/boa-dev/boa/issues/439):
+  Implement break handling in switch statements (@Lan2u)
+- [FEATURE #301](https://github.com/boa-dev/boa/issues/301):
+  Implementing the switch statement in the new parser (@Lan2u)
+- [FEATURE #120](https://github.com/boa-dev/boa/issues/120):
+  Implement `globalThis` (@zanayr)
+- [FEATURE #513](https://github.com/boa-dev/boa/issues/513):
+  Implement `Object.is()` method (@tylermorten)
+- [FEATURE #481](https://github.com/boa-dev/boa/issues/481):
+  Implement global `undefined` property (@croraf)
+
+Bug Fixes:
+
+- [BUG #412](https://github.com/boa-dev/boa/pull/412):
+  Fixed parsing if statement without else block preceded by a newline (@HalidOdat)
+- [BUG #409](https://github.com/boa-dev/boa/pull/409):
+  Fix function object constructable/callable (@HalidOdat)
+- [BUG #403](https://github.com/boa-dev/boa/issues/403)
+  `Value::to_json()` does not handle `undefined` correctly (@n14little)
+- [BUG #443](https://github.com/boa-dev/boa/issues/443):
+  HasOwnProperty should call GetOwnProperty and not GetProperty (@n14little)
+- [BUG #210](https://github.com/boa-dev/boa/issues/210):
+  builtinfun.length undefined (@Croraf)
+- [BUG #466](https://github.com/boa-dev/boa/issues/466):
+  Change `ToPrimitive()` (`to_primitive()`) hint to be an enum, instead of string (@HalidOdat)
```

Comment:
```suggestion
  Change `ToPrimitive()` (`to_primitive()`) hint to be an enum, instead of string (@Colin-Suckow)
```
I was not the one who created the PR, I just gave some advice, I don't think I should get all the credit :)